blocks/asset-results/asset-results.js

```diff
@@ -217,34 +237,32 @@ export default function decorate(block) {
     });
     const pictureDiv = modal.querySelector('.asset-results-oneup-picture');
     const moreDiv = modal.querySelector('.asset-results-oneup-more');
-    console.log(asset);
+
+    const assetDescription = asset.alt != undefined ? asset.alt : asset.caption;
+    const createdDate = asset.created != undefined ?  new Date(asset.created).toLocaleDateString() : 'N/A';
+    const modDate = asset.modified != undefined ?  new Date(asset.modified).toLocaleDateString() : 'N/A';
+
+    const displayNameMap = {
+      rum: "Website",
+      stock: "Stock",
+      brandportal: "Brand Portal",
+      aem: "AEM"
+    };
```

Comment:
This currently looks like this:

<img width="177" alt="sources" src="https://user-images.githubusercontent.com/229022/168410661-61f35da7-179f-4d44-80ad-a59ef1424e65.png">

Note that `Website` is `rum` (so far filled with blog.adobe.com Helix), but we'll also have `website` as another source type (pure crawling, no usage data) which I am filling with `www.adobe.com` stuff right now.

IMO we should not show the `sourceType` to users in the UI, but rather the `sourceDomain`. This allows to separate between multiple AEM Assets instances (granted, we just have one right now) and multiple websites (blog.adobe.com vs. www.adobe.com).

FYI, here is the sourceDomain facet from Algolia:

<img width="302" alt="sourceDomains" src="https://user-images.githubusercontent.com/229022/168410741-069b2486-ad9b-4f4e-8d89-681d61623684.png">

Reply:
I believe both facets have their merit. One way of allowing more fine-granular control would be to show the `sourceDomain` facets only when a selection has been made in the "Sources" facet and when there are more than one option.